llvm/lib/Target/AMDGPU/VOP1Instructions.td

```diff
@@ -413,10 +438,14 @@ class VOP_MOVREL<RegisterOperand Src1RC> : VOPProfile<[untyped, i32, untyped, un
 
   let HasDst = 0;
   let EmitDst = 1; // force vdst emission
+  let HasModifiers = 1;
```

Comment:
These overrides of HasModifiers , Src0Mod, Src0ModVOP3DPP should be done before overriding InsVOP3Base.